acceptance/indy_acceptance/indy-tests/test_scripts/test_scenario_base.py

```diff
@@ -0,0 +1,74 @@
+"""
+Created on Nov 22, 2017
+
+@author: khoi.ngo
+
+Containing the test base class.
+"""
+import time
+import os
+import inspect
+from libraries.utils import generate_random_string, run_async_method, make_final_result
+from libraries.constant import Constant
+from libraries.common import Common
+from libraries.logger import Logger
+from libraries.result import TestResult
+from libraries.step import Steps
+
+
+class TestScenarioBase(object):
```

Comment:
Any class extends `object`. So there is no need to specify `object` as a superclass explicitly.

Reply:
Updated. Please refer to the latest code.